stencil/src/components/pulumi-tertiary-nav/pulumi-tertiary-nav.tsx

```diff
@@ -0,0 +1,64 @@
+import { Component, Host, h, Prop } from "@stencil/core";
+
+export interface TertiaryNavItem {
+  // The anchor for the content that the nav item controls viewability of
+  anchor: string;
+  // The text that should show for the nav item
+  label: string;
+}
+
+@Component({
+    tag: "pulumi-tertiary-nav",
+    styleUrl: "pulumi-tertiary-nav.css",
+    shadow: false,
+})
+export class PulumiTertiaryNav {
+    // TertiaryNavItem[], passed into the component as stringified JSON
```

Comment:
Channeling my inner chrsmith:

```suggestion
    // TertiaryNavItem[], passed into the component as stringified JSON.
```